project/src/components/offers-sorting/offers-sorting.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { useAppDispatch, useAppSelector } from '../../hooks/useAppDispatch';
+import { setSortTypeAction } from '../../store/actions';
+import { SortType } from '../../types/types';
+
+type OffersSoringType = {
+  options: SortType[];
+}
+
+export const OffersSorting = ({options}: OffersSoringType): JSX.Element => {
+  const [isOpened, setOpened] = useState(false);
+  const dispatch = useAppDispatch();
+  const sortType = useAppSelector((state) => state.sortType);
+
+  const onSortChange = (option: SortType) => {
```

Comment:
```suggestion
  const handleSortChange = (option: SortType) => {
```

см. кридерий Д2 https://up.htmlacademy.ru/react/10/criteries#d2
надо везде проверить перед защитой правильно наименования хэндлеров